src/Helpers/Click/ClickedAppDownload.ts

```diff
@@ -0,0 +1,15 @@
+import { ActionType, ClickedAppDownload } from "../../Schema"
+
+export const clickedAppDownload = (
```

Comment:
If there is no property transformation happening in the helper you can probably just type the tracking props via the interface, this won't accomplish much besides saving us from passing the `action` field.

Reply:
This is useful because we have quite a few links around the site where we download the app, now we can just import this helper when we need to.

Reply:
This seems to just duplicate the `ClickedAppDownload` schema as a function-- I don't see the utility, also breaks the props transformation pattern from the existing helpers?